Dockerfile

```diff
@@ -1,20 +1,18 @@
-FROM java:openjdk-8
+FROM maven:3.5-jdk-8
 ENV MAXWELL_VERSION=1.10.8 KAFKA_VERSION=0.10.1.0
 
-RUN apt-get update \
-    && apt-get -y upgrade \
-    && apt-get -y install build-essential maven
-
 COPY . /workspace
 
-RUN cd /workspace \
+RUN apt-get update \
```

Comment:
Why remove `apt-get -y upgrade`? I reckon we should keep it for better security.